src/components.js

```diff
@@ -0,0 +1,44 @@
+export const UserNameInput = () => (
+  <input
+    type="text"
+    className="user-name"
+    placeholder="Please enter user name"
+  ></input>
+);
+
+export const ChangeUserButton = () => <button>Set User</button>;
+
+export const NewTodoInput = () => (
+  <input
+    type="text"
+    className="new-todo"
+    placeholder="Please enter new todo here"
+  ></input>
+);
+
+export const Header = () => (
+  <header>
+    <h1>Todo App React</h1>
+  </header>
+);
+
+export const AddTodoButton = () => <button>Add todo</button>;
+
+export const FilterTodoListButton = () => (
+  <select>
+    <option value="all">All</option>
+    <option value="completed">Completed</option>
+    <option value="outstanding">Outstanding</option>
+  </select>
+);
```

Comment:
is `FilterTodoListButton` component a button?

we could apply the same pattern passing in the list of "filters" as a prop and rendering the options in the select